### DISCUSSION TITLE
python module not found on linux

### DISCUSSION THREAD

**Discussion by Cyclopropinon** [NASA-SW-VnV/ikos#301] [Q&A]
Hello, i installed ikos on linux via brew and when trying to run it says:
```bash
$ ikos-scan
error: could not find ikos python module
error: see TROUBLESHOOTING.md
```
looking for the python packages i found:
![grafik](https://github.com/user-attachments/assets/bc320218-0a89-4702-b58a-2f9154965171)
so i did as said:
`$ export PYTHONPATH=/home/linuxbrew/.linuxbrew/Cellar/ikos/3.4_1/libexec/lib/python3.13/site-packages`
but it still wont work
i also tried it with `$ export PYTHONPATH=/home/linuxbrew/.linuxbrew/Cellar/ikos/3.4_1/libexec/lib/python3.13/site-packages/` and `$ export PYTHONPATH=/home/linuxbrew/.linuxbrew/Cellar/ikos/3.4_1/libexec/lib/python3.13/site-packages/ikos`
But it still doesnt work.
Help would be much appreciated!

**Comment by metamaterialsuit** [comment]
Did you use simply brew install? Or did you modify the brew formula? The brew formula creates a virtualenv in the setup script and sets the CMake flag -DINSTALL_PYTHON_VIRTUALENV to false.

Because I was hacking around problems with the cmake files, I hadn't noticed that, and initially had a similar problem. After I removed that line, and manually installed the virtualenv (for whatever reason, it wasn't created when I invoked Make), everything else worked.
